pkg/server/sandbox_run.go

```diff
@@ -452,6 +452,27 @@ func (c *criService) generateSandboxContainerSpec(id string, config *runtime.Pod
 	}
 	g.SetProcessOOMScoreAdj(adj)
 
+	assetAnnotations := []string{
+		annotations.KernelPath,
+		annotations.ImagePath,
+		annotations.InitrdPath,
+		annotations.HypervisorPath,
+		annotations.FirmwarePath,
+		annotations.KernelHash,
+		annotations.ImageHash,
+		annotations.InitrdHash,
+		annotations.HypervisorHash,
+		annotations.FirmwareHash,
+	}
+
+	for _, a := range assetAnnotations {
```

Comment:
maybe a helper utility that receives the arrays and adds them...

Reply:
I thought over it. If I have to move this in a separate utility then I will have to send `Generator` pointer as well as `config.Annotations` to that utility. I am not so sure if it is any benefit to making the code more modular. What do you think?

Reply:
agree more busy work than benefit :-) if we get a 2nd customer for custom annotations we'll add the helper utility then.